src/provider/provider_level_zero.c

```diff
@@ -168,8 +201,8 @@ static umf_result_t ze_memory_provider_alloc(void *provider, size_t size,
             .flags = 0};
         ze_device_mem_alloc_desc_t dev_desc = {
             .stype = ZE_STRUCTURE_TYPE_DEVICE_MEM_ALLOC_DESC,
-            .pNext = NULL,
-            .flags = 0,
+            .flags = NULL,
```

Comment:
please keep the stype, pNext, and flags initialization order. Also please set flags to 0 instead of NULL

Reply:
Done, it was a rebase conflict.